build.savant

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2014-2022, Inversoft Inc., All Rights Reserved
+ * Copyright (c) 2024, Inversoft Inc., All Rights Reserved
```

Comment:
Ideally we'd keep the existing value, so this should be `2012-2024`. If IJ is doing this, you can update the copyright config to handle it better. See the copyright config in the FusionAuth `.ipr` file. 

Same for other copyrights.